src/main/java/cambio/simulator/entities/networking/Request.java

```diff
@@ -24,7 +24,8 @@ public abstract class Request extends NamedEntity {
     private final PriorityQueue<IRequestUpdateListener> updateListeners = new PriorityQueue<>();
     private final transient Random prob;
     private MicroserviceInstance handlerInstance;
-    //microservice instance that collects dependencies of this request and computes it
+    // microservice instance that collects dependencies of this request and computes
+    // it
```

Comment:
Again, line length limit looks wierd (also in many other places in this file)
```suggestion
    // microservice instance that collects dependencies of this request and computes it
```